index.js

```diff
@@ -116,14 +120,9 @@ const walk = async (requiredDefinitions, walkFile, root, result) => {
   const definitions = await rules.reduce(reduceRules, Promise.resolve({}));
 
   if (requiredDefinitions) {
-    const validDefinitions = {};
-    Object.keys(requiredDefinitions).forEach((key) => {
-      validDefinitions[requiredDefinitions[key]] = definitions[key];
-    });
-
     result.messages.push({
       type: INNER_PLUGIN,
-      value: validDefinitions,
+      value: definitions,
```

Comment:
Changing `walk` to return all definitions, not just the required, so they can be cached and shared.